systemtest/src/main/java/io/strimzi/systemtest/logs/LogCollector.java

```diff
@@ -152,6 +180,32 @@ public synchronized void collect() {
         });
     }
 
+    private final void collectLogsForTestSuite(final Pod pod) {
+        if (pod.getMetadata().getLabels().containsKey(Constants.TEST_SUITE_NAME_LABEL)) {
+            if (pod.getMetadata().getLabels().get(Constants.TEST_SUITE_NAME_LABEL).equals(this.collectorElement.getTestClassName())) {
+                LOGGER.trace("Collecting logs for TestSuite: {}, and Pod: {}", this.collectorElement.getTestClassName(), pod.getMetadata().getName());
```

Comment:
wouldn't debug better option? OR maybe even info? Same in the bellow usages

Reply:
I could set it to `DEBUG` for sure.